2-mandatory/2-financial-times.js

```diff
@@ -1,11 +1,19 @@
 /*
     Imagine you are working on the Financial Times web site! They have a list of article titles stored in an array.
 
-    The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
+    The home page of the web site has a headline section, which only has space for article titles
+     which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
+     npm test -- --testPathPattern mandatory/2-financial-times.js
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  newArticles = [];
+  for (const article of allArticleTitles) {
```

Comment:
This is a nice implementation.
For extra practice, can you try re-writing this with the `filter` array method.

Reply:
Thank you for your review Barath.  I have made some changes as requested